pages/profile/index.md

```diff
@@ -1,11 +0,0 @@
----
-title: Your Profile
-layout: page-react.html
-entryname: profile-360
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/" onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });">Home</a></li>
-    <li><a aria-current="page" href="/profile/">Your profile</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-81364ae396faed074959382dc4922175R5.